Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'kaminari'
 gem 'soft_deletion', '~> 0.4'
 gem 'dalli', '~> 2.7.0'
 gem 'active_model_serializers', '~> 0.8.0'
+gem 'nokogiri', '1.6.7.1'
```

Comment:
isn't it better to leave a comment at least when locking to a specific version? In 6 months time nobody will know why this is here. Alternatively you could just `bundle update nokogiri` and not even change the Gemfile